src/core/constraint_template.jl

```diff
@@ -1,6 +1,7 @@
 "Limit the number of items restored in each time-step"
-function constraint_restoration_cardinality(pm::_PMs.GenericPowerModel, cumulative_repairs::Int; nw::Int=pm.cnw)
-    constraint_restoration_cardinality(pm, nw, cumulative_repairs)
+function constraint_restoration_cardinality(pm::_PMs.GenericPowerModel; nw::Int=pm.cnw)
```

Comment:
I would make this an optional keyword argument like,
```
constraint_restoration_cardinality(pm::_PMs.GenericPowerModel; nw::Int=pm.cnw, cumulative_repairs::Int=_PMs.ref(pm, nw, :repaired_total))
```
this way a user of the constraint can change the number of `cumulative_repairs` if they prefer.